src/rospub.h

```diff
@@ -24,7 +24,7 @@ NLOHMANN_JSON_SERIALIZE_ENUM(ArrayPub, {{AP_INVALID, nullptr},
 
 void init();
 void shutdown();
-void publish(const pose_t& pose, PointPub topic); // Also can be used for point_t
-void publish_array(const points_t& points, ArrayPub topic);
+void publish(const navtypes::pose_t& pose, PointPub topic); // Also can be used for point_t
+void publish_array(const navtypes::points_t& points, ArrayPub topic);
```

Comment:
Do we actually still need `rospub.cpp`/`rospub.h` anymore? They currently just implement a custom websocket protocol I made to replace ROS and bridge the gap while we were waiting for the rest of the protocol stuff to get done. As a result they have a bit of a weird interface that made more sense for ROS. Now that we're implementing PlanViz in Mission Control we should probably replace this with the appropriate Mission Control protocol methods for publishing that. We can leave it for now but at some point we should remove this

Reply:
Yeah I figured the rospub stuff is obsolete now, but I wasn't sure if it should be removed now or after the mission control interface was merged.

Reply:
Yeah, let's just leave it for now - I can remove it when I implement the PlanViz part on the rover side.